src/redirectParams.js

```diff
@@ -1,6 +1,5 @@
-import _ from 'lodash';
-import { filterState } from '@folio/stripes-components/lib/FilterGroups';
-
+import get from 'lodash/get';
+import { filterState } from '@folio/stripes/components';
```

Comment:
Not exported yet.

Reply:
In the mean time, filterState can be accessed from FilterGroups
```
import { FilterGroups } from '@folio/stripes/components';
const { filterState } = FilterGroups;
```